packages/entities/entities-plugins/src/types/plugin.ts

```diff
@@ -46,6 +53,12 @@ export enum PluginScope {
   CONSUMER_GROUP = 'consumer_group',
 }
 
+export interface PluginEntityInfo {
+  entity: PluginScope,
+  entityEndpoint: EntityType,
```

Comment:
nit: commas aren't needed
```suggestion
  entity: PluginScope
  entityEndpoint: EntityType
```